src/Tizen.Multimedia.Util/ThumbnailExtractor/ThumbnailExtractor.cs

```diff
@@ -246,7 +246,10 @@ private static Native.ThumbnailExtractCallback GetCallback(TaskCompletionSource<
         /// </summary>
         /// <remarks>
         /// The size of generated thumbnail will be 320x240.<br/>
-        /// If you want to set the size of generated thumbnail, please use <see cref="Extract(string, Size)"/><br/>
+        /// But, if the size of <paramref name="path"/> has different ratio with 320x240 (approximately 1.33:1),<br/>
+        /// thumbnail will be generated in a way to keep the ratio of <paramref name="path"/>, which based on short axis of <paramref name="path"/>.<br/>
+        /// For example, if the size of <paramref name="path"/> is 900x500 (1.8:1), the size of generated thumbnail is 432x240(1.8:1).<br/>
+        /// If you want to set the size, which is different with 320x240, please use <see cref="Extract(string, Size)"/>.<br/>
```

Comment:
Suggestion:
        /// To set the size different from 320x240, use ...

Reply:
I'll fix it.